js/lib/beautify-html.js

```diff
@@ -84,7 +84,7 @@
         return s.replace(/\s+$/g, '');
     }
 
-    function style_html(html_source, options, js_beautify, css_beautify) {
+    function style_html(html_source, options, js_beautify_raw, css_beautify_raw) {
```

Comment:
Well I don't think anything similar to this will be useful for beautifying js or css as those files will never have blocks with the other two languages as html does